portability-transfer/src/main/java/org/dataportabilityproject/transfer/JobPollingService.java

```diff
@@ -60,6 +60,8 @@ protected void runOneIteration() throws Exception {
     }
   }
 
+  // TODO: the delay should be more easily configurable
```

Comment:
Do you want this file in this PR or separate?

Reply:
Separate, I made an issue #400